### PR TITLE
Path sep

### LINKED ISSUE #36: not working in windows
using windows 7, iojs@1.5.1.
```
server.use(mount('/build', serve(__dirname)));
```
If serve = require('koa-static'), then the static files is accessible.
If serve = require('koa-static-cache'), then the static files is not accessible at all.

Here's a piece of the debug message:
```
request: /build/server.js
  koa-router GET undefined +8s
  koa-router test "/" /^(?:\/(?=$))?$/i +2ms
  koa-mount mount /jspm_packages staticCache -> false +5s
  koa-mount mount /build staticCache -> /server.js +1ms
  koa-mount enter /build/server.js -> /server.js +1ms
  koa-mount leave /build/server.js -> /server.js +2ms
```

looks like the file was found.. but somehow was not sent out, the browser request ends up in 'not found'.

### PR DESCRIPTION
**Pull request by coderhaoxin**
closes #36

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ var crypto = require('crypto')
 var fs = require('fs')
 var zlib = require('zlib')
 var path = require('path')
+var path = require('path')
```
Comment (dead-horse) [inline index.js]: already required

File: index.js [hunk 2]
```diff
@@ -15,7 +16,7 @@ module.exports = function staticCache(dir, options, files) {
   }
 
   options = options || {}
-  options.prefix = (options.prefix || '').replace(/\/$/, '') + '/'
+  options.prefix = (options.prefix || '').replace(/\/$/, '') + path.sep
```
Comment (dead-horse) [inline index.js]: `.replace(/\/$/, path.sep)`

File: index.js (hunk 1 above)
Comment (coderhaoxin) [inline index.js]: sorry

File: index.js (hunk 2 above)
Comment (coderhaoxin) [inline index.js]: done

**Comment by coderhaoxin** [comment]
wait a minute

File: index.js (hunk 2 above)
Comment (coderhaoxin) [inline index.js]: `replace(/\/|\\$/, '') + path.sep`  ?

Comment (dead-horse) [inline index.js]: `prefix` is url prefix, it shouldn't be `\\` IMO

Comment (dead-horse) [inline index.js]: sorry, it should be `.replace(/\/?$/, path.sep)`

**Comment by coderhaoxin** [comment]
@dead-horse please publish this, just forgot that I have no permission. sorry :cry:

**Comment by jonathanong** [comment]
added you!

**Comment by coderhaoxin** [comment]
published

**Pull request merged**